packages/ui/components/navigation/tabs/HorizontalTabItem.tsx

```diff
@@ -31,9 +39,14 @@ const HorizontalTabItem = function ({ name, href, linkProps, avatar, ...props }:
       href={href}
       {...linkProps}
       className={classNames(
-        isCurrent ? "bg-subtle text-emphasis" : "  hover:bg-subtle hover:text-emphasis text-default ",
+        isCurrent
+          ? isRoutingFormsTabs
+            ? "bg-emphasis text-emphasis"
+            : "bg-subtle text-emphasis"
```

Comment:
If we're only using `isRoutingFormsTabs` to change classnames let's just allow the component to receive new classnames as prop. See my previous comment.